db/migrate/20191213123801_create_whitehall_imported_assets.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class CreateWhitehallImportedAssets < ActiveRecord::Migration[6.0]
+  def change
+    create_table :whitehall_imported_assets do |t|
+      t.references :whitehall_import,
+                   foreign_key: { to_table: :whitehall_imports,
+                                  on_delete: :restrict },
+                   index: true,
+                   null: false
+
+      # Asset must be one of FileAttachment or Image
+      # @TODO - write a custom validator?
+      t.references :file_attachment_revision,
+                   foreign_key: { to_table: :file_attachment_revisions,
+                                  on_delete: :cascade }
+      t.references :image_revision,
+                   foreign_key: { to_table: :image_revisions,
+                                  on_delete: :cascade }
+
+      t.string :original_asset_url, null: false
+      t.string :variants, json: true, default: "{}"
+      t.string :state, null: false, default: "not_processed"
+      t.text :error_log
```

Comment:
Not a biggie, as I know this is consistent with whitehall_import class, but I do wonder if we're starting an usual precedent with this error_log field. Since we don't actually use it to store a log of errors and just a single one. 

Just something to think about given we're repeating it so I wonder if you share my thoughts whether we want to rename this to say "error_message" and then do same to WhitehallImport. What do you think?

Reply:
I agree that the naming of `error_log` suggests a number of entries, and that we only really intend to store one message. `error_message` would be a better name. I'll change it in this table 👍